### ISSUE TITLE
@types/express is required by server middleware 

### ISSUE THREAD

**Issue by anonyo** [honeybadger-io/honeybadger-js#907]
## What are the steps to reproduce this issue?

1. cd into https://github.com/honeybadger-io/honeybadger-js/tree/master/packages/js/examples/aws-lambda
2. yarn install
3. sls offline start

## What happens?
```
➜  aws-lambda-typescript git:(master) ✗ sls offline start

Compiling with Typescript...
Using local tsconfig.json
/Users/some_user/src/honeybadger-js/packages/js/examples/aws-lambda-typescript/node_modules/@honeybadger-io/js/dist/server/middleware.d.ts (1,49): Cannot find module 'express' or its corresponding type declarations.
Typescript compiled.
Watching typescript files...
Starting Offline: dev/us-east-1.
```

Also happening on version 
```
    "@honeybadger-io/js": "^4.3.1",
```

## What were you expecting to happen?

- Shouldn't have to add `@types/express`. 
- Be able to support applications without express. 

## Any logs, error output, etc?

```
/Users/some_user/src/honeybadger-js/packages/js/examples/aws-lambda-typescript/node_modules/@honeybadger-io/js/dist/server/middleware.d.ts (1,49): Cannot find module 'express' or its corresponding type declarations.
```

## What versions are you using?
**Operating System:** MacOS 12.6
**Package Name:** @honeybadger-io/js
**Package Version:** 4.3.1
**Browser Version:** Version 106.0.5249.103 (Official Build) (x86_64)

## Any other comments?

Is there a way to ommit `dist/server/middleware.d.ts `?

Thanks in advance!

<img src="https://front.com/assets/img/favicons/favicon-32x32.png" height="16" width="16" alt="Front logo" /> [Front conversations](https://app.frontapp.com/open/top_cnvmp)

**Comment by joshuap** [comment]
Thanks @anonyo! We're working on this one, but one of our maintainers is on vacation so it might be a few weeks. In the meantime, I think you can fix this error by adding the `@types/express` package:

```
// npm:
npm add @types/express --save

// yarn:
yarn add @types/express
```

**Comment by subzero10** [comment]
@anonyo This should be fixed with [v4.6.2](https://github.com/honeybadger-io/honeybadger-js/releases/tag/v4.6.2)!
